README.md

```diff
@@ -1,2 +1,63 @@
-# django-redis-secure
-This is a Django library that is used to encrypt data stored to Redis, and decrypt it when getting it.
+# Django secure redis
+This library provides extra option for django-redis library, in which it provides a secure layer on the top of redis cache
+
+# Installation
```

Comment:
please update to remove client and data migration, and include warning about ``incr`` and a recommendation on how to migrate data.